### DISCUSSION TITLE
Welcome to mmcalendar Discussions!

### DISCUSSION THREAD

**Discussion by chanmratekoko** [chanmratekoko/mmcalendar#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SaingHmineTun** [comment]
I saw your project support in Mon as well. What if I want to add Shan into your project? Can I?

**Reply by chanmratekoko** [reply]
@SaingHmineTun 

Certainly! Warmly welcome contributions to my project, including the addition of new languages. Adding Shan to the project sounds like a great idea.

If you have any questions or need guidance on the process, feel free to reach out to me. I appreciate your interest in enhancing my project with the inclusion of Shan. Thank you for your willingness to contribute!


**Current Language Localization Support**

1. English
2. Myanmar (Burmese Unicode) (default)
3. Myanmar (Burmese Zawgyi)
4. Myanmar (Mon)
5. Myanmar (Tai)
6. Myanmar (Karen)

Feel free to contribute to this location as well.
[Google Sheet](https://docs.google.com/spreadsheets/d/15UmLl19s3FFbctP1AGva40g_PYIA-6hvee73uNY5ZmM/edit#gid=0)
